discord/ui/button.py

```diff
@@ -260,11 +286,11 @@ def button(
 
     .. note::
 
-        Buttons with a URL cannot be created with this function.
-        Consider creating a :class:`Button` manually instead.
-        This is because buttons with a URL do not have a callback
-        associated with them since Discord does not do any processing
-        with it.
+        Premium buttons as well as buttons with a URL cannot
+        be created with this function. Consider creating a
+        :class:`Button` manually instead. This is because buttons
+        with a URL do not have a callback associated with them
+        since Discord does not do any processing with it.
```

Comment:
```suggestion
        Premium and link buttons cannot be created with this decorator. Consider 
        creating a :class:`Button` object manually instead. These types of 
        buttons do not have a callback associated since Discord doesn't handle 
        them when clicked.
```

I'm also not sure if "function" is the right word here

p.s. spacing is difficult